packages/sources/svelte/src/components/indicators/VtmnProgressbar/VtmnProgressbarLinear.svelte

```diff
@@ -69,9 +72,10 @@
     <line
       class="vtmn-progressbar_indicator"
       x1="0"
-      x2={`${indeterminate ? 100 : progress}%`}
+      x2="100%"
       y1="50%"
       y2="50%"
+      style:transform={indeterminate ? null : `translateX(${progress - 100}%)`}
```

Comment:
Not fan to et directly the style on the node.
I think it's better to set a var `--vtmnProgressbar-progress` (or other) and apply it on a style 

```css
.vtmn-progressbar_indicator {
  transform: translateX(var(--my_var, '0%'))
}
```

Reply:
Hi Antoine, thanks for the review :) 
Can you elaborate more on this please ?
I do understand that inline styling is not the best pattern in most cases, but I think that in this specific case it is very common:
- the `progress` value is a reactive prop of the component: to pass it to the CSS we will still have to inline the custom property or go through a heavier `setProperty` in JS
- Most of the progress bar uses this approach, including some famous design system. (But I understand that this kind of argument is not always a good argument ;) )

I would be happy to hear more from you on this

Reply:
Of course ! 

In fact, we got some issue with a style than the ponderation of the css property override the customClass passed on the component.

It's also a good practice in order to keep all style content on the same scope (css files).

On you svelte file you can have 

```svelte
<script>
   // [...]
</script>

<svelte:style>
    .progress_indicator {
        transform: translateX(var(--my_var, '0%'));
    }
</svelte:style>

<line
      class="vtmn-progressbar_indicator progress_indicator"
      style:transform={indeterminate ? null : `translateX(${progress - 100}%)`}
/>
```

On the second hand, have an inline style are bad for accessibility : 
https://www.nomensa.com/blog/css-inline-styles-and-why-they-are-considered-harmful-accessibility
But of course, for a transform, I think it's questionable :) 

In conclusion (for my opinion), use var instead of direct inline style : 
- Improve the ponderation of the style
- Easily overridable with className
- All the style are scope on the css file or `<svelte:style>` node